modules/social_features/social_landing_page/css/social_landing_page.section.introduction.scss

```diff
@@ -1,12 +1,7 @@
 .introduction {
   padding: 10px 0;
   background-color: #fff;
-  @media(min-width: 1200px) {
-    .paragraph--section & {
-      margin-left: calc(-100vw / 2 + #{1200px} / 2);
```

Comment:
@agami4 can we simplify this by just using: `calc(-50vw + #{600px});` ?

Reply:
@ribel, I've removed this part of the styles because this case add horizontal scroll